gensrc/thrift/Types.thrift

```diff
@@ -146,7 +146,7 @@ enum TTaskType {
     CLONE,
     STORAGE_MEDIUM_MIGRATE,
     ROLLUP,
```

Comment:
ROLLUP is deprecated too.
And why the comma here is marked as red?

Reply:
I think is Types.thrift decode has some settings. I have rewrite the comma for all TTaskType but still is red.